deprecated/stately-iso-collections/build.gradle

```diff
@@ -32,7 +31,7 @@ kotlin {
             }
         }
     }
-    wasm {
+    wasmJs {
```

Comment:
1. Should we also include `wasmWasi`?
2. Is the `browser()` and `binaries.executable()` required? Can we simplify it to have just the declaration?

Reply:
Already discuss on Slack so resolving here